openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -142,8 +149,9 @@ def _generate_mov(self, ffmpeg_path, instance, fps, no_of_frames,
             "tags": self.mov_options['tags']
         })
 
-    def _generate_thumbnail(self, ffmpeg_path, instance, source_files_pattern,
-                            staging_dir):
+    def _generate_thumbnail(
+        self, ffmpeg_args, instance, source_files_pattern, staging_dir
```

Comment:
I was wondering if we can skip passing `ffmpeg_args` altogether and do:
```py
ffmpeg_args = get_ffmpeg_tool_args("ffmpeg")
```

here?

Maybe have a `ffmpeg_extra_args=None` in case we want to add more...